src/adminPage/adminPage.component.test.tsx

```diff
@@ -10,22 +10,26 @@ import { MuiThemeProvider } from '@material-ui/core';
 import { buildTheme } from '../theming';
 import TestAuthProvider from '../authentication/testAuthProvider';
 import thunk from 'redux-thunk';
-import { act } from 'react-dom/test-utils';
-import { flushPromises } from '../setupTests';
-import {
-  loadMaintenanceState,
-  loadScheduledMaintenanceState,
-} from '../state/actions/scigateway.actions';
 import { MemoryRouter, Router } from 'react-router';
 
+import { adminRoutes } from '../state/scigateway.types';
+import { act } from 'react-dom/test-utils';
+
 describe('Admin page component', () => {
   let mount;
   let mockStore;
   let state: StateType;
+  let history: History;
+  const setStateMock = jest.fn();
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const useStateMock: any = (useState: any) => [useState, setStateMock];
```

Comment:
I'm not a fan of mocking `useState` - from what I can tell it's used to assert that the tab change is done - can we not just test that we find what we expect according to the tab e.g. on the download page we expect a div with id `datagateway-download`, on the maintenance page you can check we have the textboxes or buttons etc.